### DISCUSSION TITLE
Should we expect .NET 5 preinstalled on 20.04?

### DISCUSSION THREAD

**Discussion by piqjen-1xofFi-sikxyh** [microsoft/WSL#6298] [General] [answered]
Should we expect .NET 5 preinstalled on 20.04?

__Originally posted by @aannenko in https://github.com/actions/virtual-environments/issues/1816#issuecomment-736818040__

**Comment by Biswa96** [comment]
GitHub Actions already has Ubuntu distribution. That issue is not related to WSL.

**Comment by sirredbeard** [comment, marked as answer]
I believe your question is related to GitHub Actions. However, we have no plans to pre-install .NET 5, or any .NET, on Ubuntu WSL images at this time.
